google-cloud-storage/src/main/java/com/google/cloud/storage/UnifiedOpts.java

```diff
@@ -2288,6 +2288,22 @@ Decoder<Bucket, Bucket> clearBucketFields() {
           .orElse(Decoder.identity());
     }
 
+    <L extends T> Opts<T> prepend(Opts<L> l) {
```

Comment:
another nit; this is more just overriding values from l. could we use something like overwrite instead of prepend?

Reply:
I think `overwrite` would be confusing because the values in `l` will have a lower priority than any same type value already in `this`. Nothing passed to this method will override anything from `this`.

For example if:
```java
Opts<ListOpt> right = Opts.from(pageSize(10), delimiter(":"), userProject("proj"));
Opts<UserProject> left = Opts.from(userProject("project2"));
```

The result of calling `right.prepend(left)` would be an effective no-op since userProject is already defined in `right`. 

However if instead we had 
```java
Opts<ListOpt> right = Opts.from(pageSize(10), delimiter(":"));
Opts<UserProject> left = Opts.from(userProject("project2"));
```

The result of calling `right.prepend(left)` would be 
```java
Opts<ListOpt> result = Opts.from(pageSize(10), delimiter(":"), userProject("project2"));
```

Reply:
I think thats fair. Could you add a comment around the values in `this` having higher priority than `l` when merged? can be done in a follow up.

Reply:
I'll add a comment to the method and cleanup some variable names to be more clear.